internal/utils/utils.go

```diff
@@ -17,9 +18,101 @@ import (
 	infernoConfig "github.com/llm-inferno/optimizer-light/pkg/config"
 	"go.uber.org/zap/zapcore"
 	"gopkg.in/yaml.v3"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// Global backoff configurations
+var (
+	// Standard backoff for most operations
+	StandardBackoff = wait.Backoff{
+		Duration: 100 * time.Millisecond,
+		Factor:   2.0,
+		Jitter:   0.1,
+		Steps:    5,
+	}
+
+	// Slow backoff for operations that need more patience
+	ReconcileBackoff = wait.Backoff{
+		Duration: 500 * time.Millisecond,
+		Factor:   2.0,
+		Steps:    5,
+	}
+
+	// Prometheus validation backoff with longer intervals
+	PrometheusBackoff = wait.Backoff{
```

Comment:
This backoff is too long, can you add a TODO saying investigate why prom needs long backoff?